### ISSUE TITLE
Classification Loss: CE or BCE ? 

### ISSUE THREAD

**Issue by clw5180** [ultralytics/yolov5#720]
Hi Glenn, In ultrallytics/yolov3, you said replacing Binary Cross Entropy (BCE) loss with Cross Entropy (CE) loss significantly improves Precision, Recall and mAP, but I find BCE is better than CE, and you also use BCE now. Have you compared the BCE loss and CE loss in yolov5? (ignore the multi-label question)

**Comment by clw5180** [comment]
I see in your previous code, for CELoss there's no obj_loss, and obj_conf seems to become the conf for background ( 1 background + 80 classes  ).  Am I correct ? 
 ```
       elif 'CE' in arc:  # unified CE (1 background + 80 classes)
            t = torch.zeros_like(pi[..., 0], dtype=torch.long)  # targets
            if nb:
                t[b, a, gj, gi] = tcls[i] + 1 
            lcls += CE(pi[..., 4:].view(-1, model.nc + 1), t.view(-1))
```
